app/views/log/index.haml

```diff
@@ -1,26 +1,28 @@
 %h1 Dispatch Log
 
-= form_with model: @entry, url: log_index_url do |f|
-  .field= f.text_area :text, size: '80x8'
-  .actions= f.submit 'Create log entry'
+.row
+  .col-md-6.offset-md-3
+    = form_with model: @entry, url: log_index_path do |f|
```

Comment:
Any particular reason you didn't use bootstrap_form_with here?

Reply:
Didn't seem like as much of a benefit because no labels? But, maybe you're right.